tests/providers/amazon/aws/hooks/test_batch_waiters.py

```diff
@@ -274,19 +273,17 @@ def test_batch_job_waiting(aws_clients, aws_region, job_queue_name, job_definiti
     # moto transitions the Batch job status automatically.
 
     job_name = "test-job"
-    job_cmd = ['/bin/sh -c "for a in `seq 1 2`; do echo Hello World; sleep 0.25; done"']
```

Comment:
I removed this optional parameter for multiple reasons:

- It does not add any value to the test
- For some reasons, the command failed to run when I run pytest